### DISCUSSION TITLE
Converting All reactions into a single command

### DISCUSSION THREAD

**Discussion by PrajjwalDatir** [PrajjwalDatir/Kaoi#76] [Ideas]
Right now reactions commands are :
`&pat` , `&hug` , etc.
And there are almost 20+ reaction commands.

I think it'll be crowded once commands reach 70+ mark...
So I'm thinking about converting all the reaction commands into a single `&react` command which will have `&r` as an alias.
So all the reactions can be used as : 
`&r hug` , `&r pat` etc.

Note : I'm thinking about doing this, once number of commands get too big.
Any opinions?

**Comment by PabloChocobar69** [comment]
++ Much needed. Also for those who requires &help reaction/r gives the whole list of available reactions with the "....read more" so it doesn't spam long texts.

**Comment by anshulpurbey** [comment]
Yeah go for it 😼, can do now also...so we would be habitual to it !

**Comment by Nakshatra-14** [comment]
Yes it is good, you can do it

**Comment by Nitinkumarsahu** [comment]
Yeah... Let's try Once..

**Comment by PrajjwalDatir** [comment]
Done :v:
